firebase-functions/src/androidTest/functionsForTests/.firebaserc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Every other platform calls this directory `backend`, rather than `functionsForTests`. Could we change it, for consistency?

Reply:
done